src/RequestsLibrary/RequestsKeywords.py

```diff
@@ -57,9 +58,12 @@ def _create_session(self, alias, url, headers, cookies, auth,
         s = session = requests.Session()
         s.headers.update(headers)
         s.auth = auth if auth else s.auth
-        s.proxies = proxies if proxies else  s.proxies
-
         s.verify = self.builtin.convert_to_boolean(verify)
+        if proxies:
+            s.proxies = ast.literal_eval(proxies)
```

Comment:
Actually, yes that would work. 

I guess the question is philosophical -- add intermediate data type conversions in the test case itself or expect the library to handle it? I personally favor the latter because it makes for a cleaner test case.

That said if the current behavior is preserved, could we at least add a documentation example that demonstrates this? I had a heck of a time tracking down the problem -- it was just failing silently instead of throwing a type error.